src/main/java/be/seeseemelk/mockbukkit/inventory/meta/SkullMetaMock.java

```diff
@@ -86,12 +87,27 @@ public boolean hasOwner()
 	}
 
 	@Override
+	@Deprecated
 	public boolean setOwner(String owner)
 	{
 		this.owner = owner;
 		return true;
 	}
 
+	@Override
+	public void setPlayerProfile(com.destroystokyo.paper.profile.@Nullable PlayerProfile profile)
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
+
+	@Override
+	public com.destroystokyo.paper.profile.@Nullable PlayerProfile getPlayerProfile()
+	{
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
+	}
```

Comment:
Why was no import used here?

Reply:
I believe (but not 100% certain) that there is another PlayerProfile class from another package that is also imported.

Reply:
Yeah, we are using the org.bukkit.PlayerProfile in this class too